### ISSUE TITLE
Staking: Verification failed, connection should be dropped

### ISSUE THREAD

**Issue by tokerato** [loki-project/loki#538]
Staking from wallet 3.0.3 (lokid running locally) to service node 3.0.3 (VPS) I've successfully staked a number of service nodes from the same wallet (using version 3.0.3 / running lokid locally) - however my last service node has error:

Error: transaction <TXiD> was rejected by daemon
Error: Reason: Verification failed, connection should be dropped

On registering it asks to confirm the amount (note staking amount and fee are zero when it fails):
Staking 0.000000000 for a total fee of 0.000000000. Is this okay? 

On sweep_all:
Error: not enough outputs for specified ring size = 10:
output amount = 17.927368525, found outputs to use = 1
output amount = 17.924974688, found outputs to use = 1Please use sweep_unmixable.

On sweep_unmixable:
Error: unexpected error: Error! What is c if cols = 1!

Have tried to rescan_bc & flush transaction mempool but the error persists.

**Comment by Wysie** [comment]
Same issue here. Running SN 3.0.3 and wallet is also 3.0.3

**Comment by Wysie** [comment]
I'm running the "register_service_node"  command on my local wallet, that I get after running "prepare_registration" on the service node, and the immediate message(s) is as follows:

`Wallet password: 
Transaction 1/1:
Staking 0.000000000 for a total fee of 0.000000000. Is this okay? (Y/Yes/N/No): Y
Error: transaction <da8af651babcb7c244e6a0603f224875257aa539d16af4465fe9259ee2d1bd93> was rejected by daemon
Error: Reason: Verification failed, connection should be dropped
Error: There was an error, which could mean the node may be trying to get you to retry creating a transaction, and zero in on which outputs you own. Or it could be a bona fide error. It may be prudent to disconnect from this node, and not try to send a transaction immediately. Alternatively, connect to another node so the original node cannot correlate information.`

If I enter "N" it fails as well (obviously). Meanwhile, on my local daemon, I get the following error:

`2019-03-30 10:36:39.215	[RPC0]	WARNING	daemon.rpc	src/rpc/core_rpc_server.cpp:751	[on_send_raw_tx]: tx verification failedVerification failed, connection should be dropped
`

**Comment by Wysie** [comment]
Another update, I tried with a remote node as well, same issue on the wallet.

**Comment by Wysie** [comment]
Last update on my end, my sweep_all worked (I didn't try it earlier), and things are fine now it seems.

**Comment by KeeJef** [comment]
We have found what the issue is here, it has to do with Service Node rewards being mined in Version 1 blocks, which are being produced by a small minority of miners (We have warned the affected pools to stop creating v1 txes as their mining rewards will also not be spendable), we should have a patch in shortly which should allow you to use sweep_all properly.

We have tracked down 29 blocks so far which include v1 txes. Unfortunately these blocks also contain Service Node rewards. Those rewards total about ~900 Loki, currently those rewards are unspendable due to the fact that they cannot be mixed with other coins since they are non ring CT transactions and the Loki chain never included non ring ct txes. We will include a fix in the next hardfork to make these coins spendable.
